lib/scheduler.pm

```diff
@@ -0,0 +1,82 @@
+# Copyright © 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package scheduler;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use warnings;
+
+use File::Basename;
+use testapi qw(get_var set_var);
+use main_common 'loadtest';
+use YAML::Tiny;
+
+# use lib '/usr/share/openqa/lib';
+# use OpenQA::Utils qw(asset_type_from_setting locate_asset);
+
+our @EXPORT = qw(load_yaml_schedule);
+
+sub parse_vars {
+    my ($schedule) = shift;
+    my %vars;
+    while (my ($var, $value) = each %{$schedule->{vars}}) {
+        $value =~ s/%(.*?)%/get_var($1)/eg;
+        # my $type = asset_type_from_setting($var, $value);
+        # $value = locate_asset($type, $value) if $type;
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit